_posts/en/newsletters/2024-05-15-newsletter.md

```diff
@@ -0,0 +1,183 @@
+---
+title: 'Bitcoin Optech Newsletter #302'
+permalink: /en/newsletters/2024/05/15/
+name: 2024-05-15-newsletter
+slug: 2024-05-15-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Release of utreexod beta:** Calvin Kim [posted][kim utreexo] to the
+  Bitcoin-Dev mailing list to announce the beta release of `utreexod`, a
+  full node with support for [utreexo][topic utreexo].  Utreexo allows a
+  node to store a small commitment to the state of the UTXO set rather
+  than the entire set itself; for example, a minimal commitment can be
+  32 bytes and the current full set is about 12 GB, making the
+  commitment on the order of about a billion times smaller.  To reduce
+  bandwidth, utreexo may store additional commitments, increasing its
+  use of disk space, but still keeping its chainstate roughly on the
+  order of a million times smaller than a traditional full node.
+  A utreexo node that also prunes old blocks can run in small constant
+  amount of disk space, whereas even pruned regular full nodes can have
+  their chainstate grow beyond the bounds of a device's storage capacity.
+
+  The release notes posted by Kim indicate that the node is compatible
+  with [BDK][bdk repo]-based wallets, plus many other wallets through
+  support for [Electrum personal server][].  The node supports
+  transaction relay with extensions to the P2P network protocol for
+  allowing relay of utreexo proofs.  Both _regular_ and _bridge_ utreexo
+  nodes are supported; regular utreexo nodes use the utreexo commitment
+  to save disk space; bridge nodes store the complete UTXO state plus
+  some additional data and can attach utreexo proofs to blocks and
+  transactions created by nodes and wallets that don't support utreexo
+  yet.
+
+  Utreexo does not require any consensus changes and utreexo nodes do
```

Comment:
```suggestion
  Utreexo requires no consensus changes and utreexo nodes do
```
Or, "Utreexo does not require consensus changes" (the word "any" seems like overly-convincing language).